newIDE/app/src/AssetStore/AssetCard.js

```diff
@@ -19,6 +22,11 @@ const styles = {
     verticalAlign: 'middle',
     pointerEvents: 'none',
   },
+  previewImagePixelated: {
+    width: '100%',
+    imageRendering: '-moz-crisp-edges' || 'pixelated', // TODO to check
```

Comment:
Can we just use crisp-edges? Seems supported on all our targets:
![image](https://user-images.githubusercontent.com/1280130/127879692-35906a94-8e09-45e0-bc6b-b4bcff46654d.png)

Reply:
The table from MDN and my tests are different.

![image](https://user-images.githubusercontent.com/1670670/128344741-fbcf861e-3696-48de-8192-0c5e3ab14dc8.png)

Reply:
🤯
Alright then, let's do a helper with CSS.supports:

```
// @flow
let pixelatedImageRenderingValue: ?string = null;
export const getPixelatedImageRendering = (): string => {
  if (pixelatedImageRenderingValue) return pixelatedImageRenderingValue;
  
  return (pixelatedImageRenderingValue = CSS.supports('image-rendering: crisp-edges') ? 'crisp-edges' : 'pixelated');
}
```

In a CSSHelper.js file in Utils and then call it: `imageRendering: getPixelatedImageRendering(),` and we should be good (I cache the result to avoid calling CSS.supports at every render).